### ISSUE TITLE
About VGGT training details

### ISSUE THREAD

**Issue by jacky-hjqq** [facebookresearch/vggt#194]
Hello, I want to train vggt on my custom data, and find that during training, the first frame always has better depth and pts prediction and other frames are very bad. So I am curious if you also have such observation when train VGGT that first frame is easier to learn and other frames are harder. 

And also could you tell me the batch size used for training? Thanks!

**Comment by daiyixiang666** [comment]
Are you using the loss from the training branch?  I also find this problem.  The loss from the remaining frames are much higher than the first frame even in the overfitting testing

**Comment by jacky-hjqq** [comment]
Yes i use the loss from the traning branch, and i tried in the overfitting, the first frame can be learnt very fast and converg. The other frames can't be learnt and converg at all.

**Comment by daiyixiang666** [comment]
Do you try do not predict the conf ?

**Comment by jacky-hjqq** [comment]
yes I set the "disable_conf=True"

**Comment by daiyixiang666** [comment]
How about your overfitting result in one sample? Is this normal?

**Comment by jacky-hjqq** [comment]
On one sample, also the first frame is much better. What about you?

**Comment by jytime** [comment]
Hi, yes, in the early stage of training, the result of the first frame will be much better. In the later stage, the result of the first frame should be about the same as other frames, just slightly better.

**Comment by daiyixiang666** [comment]
Hi, author, when I try to overfit on one sample, I find out that if I use the the torch.norm like the l2 norm for the sample. It converge fast, but when I use the code that you provide, it will converge very slow and also the loss will go to minus 0. Is that normal? (I know it is because of the Conf). I just kind of confused

**Comment by jytime** [comment]
It is normal. We also found L2 is faster for convergence but not good at training stability.

**Comment by daiyixiang666** [comment]
Do you try first using the L2 then using the L1 ?
